types/google.accounts/index.d.ts

```diff
@@ -45,25 +45,71 @@ declare namespace google.accounts {
         }
 
         interface OverridableTokenClientConfig {
+            /**
+             * Optional. A space-delimited list of scopes that identify the
+             * resources that your application could access on the user's
+             * behalf. These values inform the consent screen that Google
+             * displays to the user.
+             */
+            scope?: string;
+
+            /**
+             * Optional, defaults to true.
+             * Enables applications to use incremental authorization
+             * to request access to additional scopes in context.
+             * If you set this parameter's value to false and the
+             * authorization request is granted, then the new access token
+             * will only cover any scopes to which the scope requested
+             * in this TokenClientConfig.
+             */
+            include_granted_scopes?: boolean;
+
             /**
              * Optional. A space-delimited, case-sensitive list of prompts to
              * present the user.
              */
             prompt?: string;
 
             /**
-             * Optional. If set to false, more granular Google Account
-             * permissions will be disabled for clients created before 2019. No
-             * effect for newer clients, since more granular permissions is
-             * always enabled for them.
+             * Optional, defaults to true. If set to false, more granular Google
+             * Account permissions would be disabled for OAuth client IDs
+             * created before 2019. If both enable_granular_consent and
+             * enable_serial_consent are set, only enable_granular_consent value
+             * would take effect and enable_serial_consent value would be
+             * ignored.
+             *
+             * No effect for newer OAuth client IDs, since more granular
+             * permissions is always enabled for them.
+             */
+            enable_granular_consent?: boolean;
+
+            /**
+             * Deprecated, you should use enable_granular_consent instead.
+             * Optional, defaults to true.
```

Comment:
is there a source for this default? Given its deprecation I would imagine it defaults to being unset/undefined/null

Reply:
Right, when both of these settings aren't set, the optional value of `enable_granular_consent` takes precedence. My rationale was to leave the old description for older codebases, but I will update the comments to match the current Google documentation so there is no confusion.